bigtable/admin.go

```diff
@@ -247,12 +314,12 @@ type FamilyInfo struct {
 	GCPolicy string
 }
 
-// TableInfo retrieves information about a table.
-func (ac *AdminClient) TableInfo(ctx context.Context, table string) (*TableInfo, error) {
+func (ac *AdminClient) getTable(ctx context.Context, table string) (*btapb.Table, error) {
 	ctx = mergeOutgoingMetadata(ctx, ac.md)
 	prefix := ac.instancePrefix()
 	req := &btapb.GetTableRequest{
 		Name: prefix + "/tables/" + table,
+		View: btapb.Table_FULL,
```

Comment:
Is there a performance hit from getting all fields? I guess it probably doesn't matter for an admin feature.

Reply:
There may be a small hit here, but we don't expose the ability to change view levels at the moment. I could refactor this to use different view levels for different methods, but not sure if that is worth it. @igorbernstein2 any thoughts here on how expensive using Table_FULL is? Before this was using SCHEMA VIEW (default) but think I need more to get encryption information (may be able to go lower than FULL though)

Reply:
altered to narrow scope. This likely wasn't too expensive, but I liked the idea and ran with it.